public/javascripts/SVValidate/src/label/LabelVisibilityControl.js

```diff
@@ -31,16 +31,33 @@ function LabelVisibilityControl () {
      * @param {boolean} newLabel Indicates whether we unhide due to showing a new label vs. clicking the unhide button.
      */
     function unhideLabel (newLabel) {
+        let htmlString;
         let panomarker = svv.panorama.getPanomarker();
         let label = svv.panorama.getCurrentLabel();
         panomarker.setIcon(label.getIconUrl());
         panomarker.draw();
         visible = true;
-        let htmlString = `<u>H</u>ide Label</button>`;
-        labelVisibilityButtonOnPano.html(htmlString);
-        htmlString = `<img src="assets/javascripts/SVValidate/img/HideLabel.svg" class="label-visibility-control-button-icon" alt="Hide Label">
-        <br /><u>H</u>ide Label</button>`;
-        labelVisibilityControlButton.html(htmlString);
+        if (labelVisibilityButtonOnPano.text() === 'Toon Label') {
```

Comment:
Can you take a look at how we do translations elsewhere in the code, and use those methods instead? We have a JavaScript library that we use to more easily incorporate translations!